features/missing.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Missing
+  List missing dependencies
+
+  Scenario: No dependencies
+    Given this Cask file:
+    """
+    """
+    When I run cask "missing"
+    Then I should see command output:
+    """
+    """
```

Comment:
Same here as above. It will always be true.

When writing specs. Make sure they fail without the changes in the code. Otherwise you can never trust that the test is correctly implemented.